doc/asciidoc/similarity-euclidean.adoc

```diff
@@ -84,24 +84,24 @@ include::scripts/similarity-euclidean.cypher[tag=stream]
 [opts="header"]
 |===
 | `from`     | `to`       | `similarity`
-| Arya     | Karin    | 7.681145747868608
-| Zhen     | Michael  | 7.874007874011811
-| Zhen     | Praveena | 12.569805089976535
-| Praveena | Michael  | 12.727922061357855
-| Michael  | Karin    | 15.033296378372908
-| Praveena | Karin    | 16.1245154965971
-| Zhen     | Karin    | 16.30950643030009
-| Praveena | Arya     | 16.76305461424021
-| Michael  | Arya     | 17.406895185529212
-| Zhen     | Arya     | 19.621416870348583
+| "Zhen"     | "Arya"     | 0.0
+| "Zhen"     | "Karin"    | 0.0
+| "Praveena" | "Karin"    | 3.0
+| "Zhen"     | "Michael"  | 3.605551275463989
+| "Praveena" | "Michael"  | 4.0
+| "Arya"     | "Karin"    | 4.358898943540674
+| "Michael"  | "Arya"     | 5.0
+| "Zhen"     | "Praveena" | 6.708203932499369
+| "Michael"  | "Karin"    | 7.0
+| "Praveena" | "Arya"     | 8.0
 
 |===
 // end::stream[]
 
-Arya and Karin have the most similar food preferences, with a euclidean distance of 7.68.
+Zhen and Arya and Zhen and Karin have the most similar food preferences, with a euclidean distance of 0.0.
```

Comment:
this sentence needs a comma:

Zhen and Arya, and Zhen and Karin have the most similar food preferences...